### PR TITLE
Convert javascript into an AMD module

### PR DESCRIPTION
**Pull request by juho-jaakkola**

### CODE REVIEW

File: views/default/input/granular_access.php [hunk 1]
```diff
@@ -28,13 +30,14 @@
...
-		
-		if (array_search($vars['value'], $options_values) === false) {
-			$set_custom = true;
-		}
+	} else {
+		$options_values = get_write_access_array();
+	}
+
+	if (array_search($vars['value'], $options_values) === false) {
+		$set_custom = true;
```
Comment (juho-jaakkola) [inline views/default/input/granular_access.php]: Changes to this part aren't really related to the javascript. I couldn't however get the feature to work without this fix.

I can make another PR for this change, if you verify that the fix is correct, and you'd rather have it as a separate commit.

Comment (beck24) [inline views/default/input/granular_access.php]: No prob, I'm not that picky with my own repos :)

**Comment by beck24** [comment]
LGTM

**Comment by juho-jaakkola** [comment]
Dammit, I did the PR for the wrong branch. Well, technically you could still force push the current master into `1.9` branch.

**Comment by juho-jaakkola** [comment]
Actually it doesn't even need `--force` because the history is identical.

**Comment by beck24** [comment]
oh, I assumed you were going for master, one sec

**Comment by beck24** [comment]
ok, should be good now

**Pull request merged**